packages/web/src/features/translation/TranslateWord.tsx

```diff
@@ -107,6 +112,14 @@ const TranslateWord = forwardRef<TranslateWordRef, TranslateWordProps>(
           <span className="inline-block" ref={ancientWord}>
```

Comment:
Can we move the tabIndex and click event to this span instead? Then we can call `onShowDetail` from the button click as well as `onOpenNotes`. This is a bit more explicit as to what is different about each click region.

Reply:
Done.